app/uk/gov/hmrc/transitmovements/models/responses/DeclarationResponse.scala

```diff
@@ -18,12 +18,12 @@ package uk.gov.hmrc.transitmovements.models.responses
 
 import play.api.libs.json.Json
 import play.api.libs.json.OFormat
-import uk.gov.hmrc.transitmovements.models.DepartureId
 import uk.gov.hmrc.transitmovements.models.MessageId
+import uk.gov.hmrc.transitmovements.models.MovementId
 import uk.gov.hmrc.transitmovements.models.formats.PresentationFormats
 
 object DeclarationResponse extends PresentationFormats {
   implicit val declarationResponseFormat: OFormat[DeclarationResponse] = Json.format[DeclarationResponse]
 }
 
-case class DeclarationResponse(departureId: DepartureId, messageId: MessageId)
+case class DeclarationResponse(movementId: MovementId, messageId: MessageId)
```

Comment:
This should remain `departureId` because that's what the API expects for a declaration response.

Reply:
ok, updated to use departureId